test/puppetlabs/comidi_test.clj

```diff
@@ -88,7 +54,7 @@
                       [["/bar/" :bar]
                        [["/baz" {:get :baz-handler}]
                         ["/bam" {:put :bam-handler}]
-                        ["/bap" {:any :bap-handler}]]]
+                        ["/bap" {:options :bap-handler}]]]
```

Comment:
```:options``` isn't a valid ```Pattern``` as far as I can see:

https://github.com/juxt/bidi/blob/master/src/bidi/schema.cljx#L37

I figured that was intended as a ```MethodGuard``` so chose another we aren't using. A keyword in a vector would be fine though e.g. ```["foo]```